src/components/DialogFooter/DialogFooter.tsx

```diff
@@ -0,0 +1,43 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { ReactElement } from 'react';
+import { Button, ButtonEmphasis, ButtonSize, ButtonStyle } from '@components/Button';
+
+export type DialogButton = {
+    label: string;
+    action: () => void;
+};
+
+export type DialogFooterProps = {
+    buttons: [DialogButton] | [DialogButton, DialogButton];
```

Comment:
Any reason why cant this just be a `DialogButton[]`? I know `cancel` and `confirm` are used in 99% of cases, but I don't feel like it would hurt anything. And then just do `buttons.map(...)`.
This is just a nitpick, but I love the direction it's going to for now 😄

Reply:
Yes, It was a design decision to limit to only 2 but they weren't against having more. In this case, I would need to give the option to choose the styling of the button. 🤔